install-deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```

Comment:
On 22-3-2016 04:07, Kefu Chai wrote:
> In install-deps.sh
> <https://github.com/ceph/ceph/pull/7515#discussion_r56931889>:
> 
>> @@ -1,4 +1,4 @@
>> -#!/bin/bash -e
>> +#!/bin/sh -e
> 
> how's your test going? i see you instruct user to create a symlink in
> README.FreeBSD, so we can dispense with this change?

Boy, you've done some serious reviewing ....
Thanx for that.

--WjW